tests/ui/parser/ranges-precedence.rs

```diff
@@ -1,7 +1,8 @@
 //@ run-pass
+//@ edition: 2021
```

Comment:
Question: should this receive revisions to test both 2015 and 2021?

Reply:
I don't think so. Nothing tested here is sensitive to edition. I added this because otherwise I have to write `extern crate core;` (2015 is a silly default).

Reply:
For the record, most UI tests just import from std for that reason.